dll/shellext/ntobjshex/regfolder.cpp

```diff
@@ -12,7 +12,7 @@
 const GUID CLSID_RegistryFolder = { 0x1c6d6e08, 0x2332, 0x4a7b, { 0xa9, 0x4d, 0x64, 0x32, 0xdb, 0x2b, 0x5a, 0xe6 } };
 
 // {18A4B504-F6D8-4D8A-8661-6296514C2CF0}
-static const GUID GUID_RegistryColumns = { 0x18a4b504, 0xf6d8, 0x4d8a, { 0x86, 0x61, 0x62, 0x96, 0x51, 0x4c, 0x2c, 0xf0 } };
+//static const GUID GUID_RegistryColumns = { 0x18a4b504, 0xf6d8, 0x4d8a, { 0x86, 0x61, 0x62, 0x96, 0x51, 0x4c, 0x2c, 0xf0 } };
```

Comment:
Keeping it as a comment makes sense, its the only reference in our code.
lgtm

Reply:
I think this is the only reference because this GUID is owned by ntobjshex, i.e. David generated it. @gigaherz any thoughts on whether this is needed?